sources/packages/web/src/views/aest/AppAEST.vue

```diff
@@ -1,45 +1,61 @@
 <template>
   <IdleTimeChecker :clientIdType="ClientIdType.AEST">
-    <v-app-bar dense flat app style="overflow: visible">
-      <BCLogo
+    <v-app-bar color="white">
+      <b-c-logo
         subtitle="Ministry of Advanced Education and Skills Training"
-      ></BCLogo>
-      <v-spacer></v-spacer>
-      <check-permission-role :role="Role.AESTCreateInstitution">
-        <template #="{ notAllowed }">
-          <v-btn
-            v-if="isAuthenticated"
-            variant="text"
-            prepend-icon="fa:fa fa-edit"
-            :disabled="notAllowed"
-            @click="
-              $router.push({ name: AESTRoutesConst.INSTITUTION_PROFILE_CREATE })
-            "
-            >Create institution</v-btn
-          >
-        </template>
-      </check-permission-role>
-      <v-menu v-if="isAuthenticated">
-        <template v-slot:activator="{ props }">
-          <v-btn
-            class="mr-5"
-            icon="fa:fa fa-user"
-            variant="outlined"
-            elevation="1"
-            color="secondary"
-            v-bind="props"
-          ></v-btn>
-        </template>
-        <v-list>
-          <v-list-item :value="menuItem.label">
-            <v-list-item-title>
-              <v-list-item-title @click="menuItem.command">
-                <span class="label-bold">{{ menuItem.label }}</span>
+      ></b-c-logo>
+      <v-btn-toggle
+        selected-class="active-btn label-bold"
+        v-model="toggleNav"
+        class="navigation-btn float-left"
+      >
+        <check-permission-role :role="Role.AESTCreateInstitution">
+          <template #="{ notAllowed }">
+            <v-btn
+              v-if="isAuthenticated"
+              class="nav-item-label"
+              variant="text"
+              :to="{
+                name: AESTRoutesConst.AEST_DASHBOARD,
+              }"
+              >Home</v-btn
+            >
+            <v-btn
+              v-if="isAuthenticated"
```

Comment:
Why is the user icon button changed to square button instead of circle ? 
![image](https://user-images.githubusercontent.com/54600590/197839133-998af1fd-3cd1-4ff8-b845-5fb1ff96e818.png)

![image](https://user-images.githubusercontent.com/54600590/197839239-ed36f2c0-915c-4395-aeab-d0086e64cb39.png)